aiojenkins/jenkins.py

```diff
@@ -18,11 +18,19 @@
 from .plugins import Plugins
 from .views import Views
 
+
 JenkinsVersion = NamedTuple(
-    'JenkinsVersion', [('major', int), ('minor', int), ('patch', int)]
+    "JenkinsVersion",
+    [("major", int), ("minor", int), ("patch", int), ("build", int)],
```

Comment:
single quotes please

Reply:
Any plans are there to enable black?

Reply:
I've created an issue about adding black here to check such type of problems. 
https://github.com/pbelskiy/aiojenkins/issues/30